cpp/tests/utilities/column_wrapper.cuh

```diff
@@ -386,8 +385,12 @@ struct column_wrapper {
     }
   }
 
-  rmm::device_vector<ColumnType> data;
-  rmm::device_vector<gdf_valid_type> bitmask;
+  rmm::device_vector<ColumnType> data;  ///< Container for the column's data
```

Comment:
This would prevent using the same class in testing for wrapping host-side columns. Have you considered using a template parameter for the container of `data` (and of `bitmask`)?

Reply:
While this is true, `column_wrapper` is really meant to be a wrapper for `gdf_column`, whose data is always assumed to be on device. 

What you're suggesting is possible, but would require a more complicated design. For example, `operator==` takes advantage of the fact that it's always wrapper device-columns by using `thrust::all_of` to check for equality. If `column_wrapper` were both a host-side and device-side abstraction, I would need to check whether its a host-side and device-side column and provide separate code paths for each option. 

Furthermore, I understand it would be a good design to do what you suggest, but I'm not sure I see the value of doing so right now. None of the `libcudf` APIs support host-side columns. In fact, if you tried to pass a host-side column into any libcudf API, it would certainly break quite spectacularly.  

I'll file this away as a `TODO` for if and when it makes sense.

Reply:
Fair enough.